app_dart/lib/src/request_handlers/vacuum_github_commits.dart

```diff
@@ -65,7 +65,10 @@ class VacuumGithubCommits extends ApiRequestHandler<Body> {
     final DateTime queryAfter = DateTime.now().subtract(const Duration(days: 1));
     final DateTime queryBefore = DateTime.now().subtract(const Duration(minutes: 3));
     try {
+      log.debug(
+          'Listing commit for slug: $slug branch: $branch and msSinceEpoch: ${queryAfter.millisecondsSinceEpoch}');
       commits = await githubService.listCommits(slug, branch, queryAfter.millisecondsSinceEpoch);
+      commits = commits ?? <RepositoryCommit>[];
```

Comment:
Is this ok to be empty, will it cause issues below?

Reply:
As long as it is not null it will prevent errors below. We still need to fix the root case which is failures in the github requests.